### DISCUSSION TITLE
Apache flies crazy when trying hover in Syria map

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#2150] [Q&A] [answered]
In the Syria map, AI George pilot or player are unable to hover. The helo just goes crazy rotating and swinging like a pendulum and unable to get off ground if it touches the ground. Cruising is ok. But hovering just goes nuts. Always getting rotar rpm low. Even George can't fly properly. Is this a known bug?

**Comment by Starfire13** [comment, marked as answer]
It's not a bug. Most likely due to high temperature. Apache performance is very affected by temperature (and height above sea level). Pick one of the winter months.

**Reply by GrEaSeLiTeNiN** [reply]
But I don't have this problem in Instant Mission Syria. In fact it's the first time I've encountered this problem.  Even AI George cannot fly the Apache and keeps crashing.

**Reply by charliefoxtwo** [reply]
I checked the instant action missions.

In the hot start mission, free air temperature is 30C and altitude is 200ft MSL
In the weapon range mission, free air temperature is 28C and altitude is 2150ft MSL

Which instant action are you talking about? These are very different scenarios.

Helicopter performance (like every aircraft) is dependent upon FAT, MSL, and weight. For example, according to the charts, in 30C FAT at 2500ft MSL and 18000lb gross weight, it will take 100% engine torque to hold a 5ft IGE hover (assuming no wind and a level surface).

In order to best help, I think we need the following information:
 - FAT of the mission
 - MSL of the location the issue occurs (I'm assuming IGE; OGE requires even more torque)
 - Gross weight of the aircraft in mission configuration

Otherwise, I'm partial to @Starfire13 's answer - probably not a bug. If you want to fly in the summer months, I suggest dropping your fuel load or performing a rolling takeoff.

**Reply by Starfire13** [reply]
If you drop fuel to 50%, you will still have over an hour of flight time with a full combat load (8 hellfires 2 rocket pods) in 40 degrees at 5700 feet (the highest altitude airfield in DCS right now). So, dropping fuel is the most effective way. The AI Apaches will actually just sit on the ground and not take off properly at 100% fuel when the weather's too hot.

**Reply by GrEaSeLiTeNiN** [reply]
Yes you are right. The Syria map ops area for Instant Action was at 20ft while for Liberation it was at 2000ft. Changing the month to Dec and temp to 10degC worked. Thanks! Is there a way to change the ongoing campaign date from Jun to Dec?
